boards/arm/ks_zion/ks_zion.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: ks_zion
+name: Kernelspace zion
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 12
+supported:
+  - gpio
+  - i2c
+  - spi
+  - can
+  - pwm
+  - dma
+vendor: st
```

Comment:
I think it should be consistent with the DTS and be "ks" but I will let @erwango confirm

Reply:
No, it isn't ST. Please fix